packages/core/lib/generators/integrations/vector_graphics_integration.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:flutter_gen_core/generators/integrations/integration.dart';
+import 'package:flutter_gen_core/generators/integrations/svg_integration.dart';
+import 'package:flutter_gen_core/settings/asset_type.dart';
+
+/// An implementation of [Integration] for `flutter_svg` and `vector_graphics`.
+///
+/// This integration depends on [SvgIntegration].
+class VectorGraphicsIntegration extends Integration {
```

Comment:
Could this integration move into the SVG's?

Reply:
Technically, possible. But, in that case, SvgGenImage and SvgVecGenImage will be always added together.

Reply:
So we can use `SvgGenImage` all the time rather than separating them?

Reply:
@AlexV525 
If it is what you really want, I'll address this.
However, it will lead to generate `SvgVecGenImage` class even if there is no `.vec` file specification in `pubspec.yaml` and the same thing happens for `SvgGenImage` as well.

Even for now, we can use `SvgGenImage` and `SvgVecGenImage` at the same time. Please see `packages/core/test_resources/pubspec_assets_vector_graphics_integrations.yaml` and `packages/core/test_resources/pubspec_assets_svg_integrations.yaml`. As long as `flutter_svg: true` is defined in `pubspec.yaml`, the users can use both of them without caring of `VectorGraphicsIntegration` and `SvgIntegration` are unified or separate. And, please notice that `VectorGraphicsIntegration.isSupported()` and `SvgIntegration.isSupported()` are currently not the same.

Reply:
What I meant is adding the integration as an extension of the SVG's, rather than a standalone integration.

Reply:
@powdream Any problem with the above suggestion?

Reply:
@AlexV525 Let me close this PR since I guess it would not work as you want.

Reply:
If you'd prefer, I can continue to work with your request's branch and see if we can match everyone's point.